src/errors.ts

```diff
@@ -62,7 +65,8 @@ export function wrapApiError(maybeError: Error, message?: string): Error {
   }
   const {data, status} = res;
   let msg = `${prefix}API responded with status ${status}`;
-  const causeMsg = typeof data == 'string' ? data : data?.error?.message;
+  const causeMsg =
+    typeof data == 'string' ? data : (data as any)?.error?.message;
```

Comment:
adding `as any` fixes the complaint but i'm not sure if this is what we want to do here. not familiar with `response.data` should look like

Reply:
what's the new `data` type look like?

Reply:
I would rather avoid casting into `any` since it might blow up at runtime

Reply:
does using strict equality '===' help the compiler here?

Reply:
let me try but i dont think so?

Reply:
i started to think it's an typo.
it should be just `error.message` instead of `data.error.message`.

Reply:
@ypc-faros ^^ can you take a look?

Reply:
i see this in Ted's PR/repo https://github.com/faros-ai/clio/blob/master/cli/src/utils.ts#L155
so i guess it's not a typo